scalableminds/webknossos
# time tracking api returns time spans with decimal places

### Mailable description of changes:
 - Time tracking api now returns decimal places instead of rounding down to seconds. Formatted like "PT0H9M9.3S"

### Steps to test:
- trace some, then watch network tab on time tracking view, new format should be returned in json
- time tracking view in frontend should still work as expected

### Issues:
- fixes #2333

------
